modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/DisplayContextUtil.java

```diff
@@ -162,12 +163,10 @@ public static Map<Long, String> getTypeNames(
 		return typeNames;
 	}
 
-	public static List<String> getTypeNamesBySite(
+	public static List<Map<String, String>> getTypeNamesBySite(
```

Comment:
We can include the count here so that we don't need to iterate through the list and map multiple times. You can use `Map<Long, ObjectValuePair<String, int>>` where 
- map key is the modelClassNameId
- ObjectValuePair key is the typeName and int is the count